Sources/Controllers/MessagesViewController+DataSource.swift

```diff
@@ -0,0 +1,99 @@
+/*
+ MIT License
+
+ Copyright (c) 2017 MessageKit
```

Comment:
2017-2018

Reply:
I copied this from an existing file. This means we missed something somewhere 😂

Reply:
Actually, it looks like I didn't merge `master` back into `development`